configuration_manager/config_logger.py

```diff
@@ -0,0 +1,54 @@
+import sys
+import os
+import logging
+import logging.config
+from logging.handlers import TimedRotatingFileHandler
+from logging.handlers import RotatingFileHandler
+
+from utils import util
+from functools import reduce
+import operator
```

Comment:
This class skeleton looks very neat. But I do not have experience with some of these packages. 
Maybe in future commits with more specific content and documentation I can give a review.